nextcloud-startup-script.sh

```diff
@@ -239,15 +239,21 @@ print_text_in_color "$ICyan" "Getting scripts from GitHub to be able to run the
 download_script LETS_ENC activate-tls
 download_script STATIC temporary-fix
 download_script STATIC update
-download_script STATIC trusted
 download_script STATIC setup_secure_permissions_nextcloud
 download_script STATIC change_db_pass
 download_script STATIC nextcloud
 download_script STATIC update-config
+# download_script NETWORK update-config
+download_script STATIC trusted
+# download_script NETWORK trusted
 download_script STATIC menu
+# download_script MENU menu
 download_script STATIC server_configuration
+# download_script MENU server_configuration
 download_script STATIC nextcloud_configuration
+# download_script MENU nextcloud_configuration
 download_script APP additional_apps
+# download_script MENU additional_apps
```

Comment:
Wouldn't it make sense to uncomment the new version and comment the old?

Reply:
Yes, it would. :smile: Or maybe the functions wants to be nostalgic about where they came from? Haha :)

Reply:
Duuh, yes it would. Thought that I was smart here adding backwards compatibility.

Thing is that the download function exits if the script doesn't exist, so I've got several emails regarding test_connection and activate-tls since those are "missing" in the old VMs (the startup script).

That was what I was trying to solve here, thinking that we could use the old references just until we make the new release.

But you're right, it doesn't make any difference right now. What actually matter is  that the scripts are at the old locations.